test/smoke/dapi.js

```diff
@@ -1,10 +1,38 @@
-describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+const DAPIClient = require('@dashevo/dapi-client');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
 
-  // getBestBlockHeight
-  it('should respond data from insight');
+const networkConfig = getNetworkConfig();
 
-  // fetchDapObjects any dapiId and type
-  it('should respond data from drive');
+networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+  const options = {
+    debug: false,
+    verbose: false,
+    errors: false,
+    warnings: false,
+    seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
+  };
+  describe(`DAPI ${nodeName}`, () => {
+    // getBlockHash height:1
+    it('should respond data from chain', async () => {
```

Comment:
How do you think should we compare obtained data with data from DC just to make sure that DAPI connected well to correct node?

Reply:
I guess that dashCore.'All Nodes' verify consistency of this data

Reply:
Hm, right.